.changeset/honest-onions-vanish.md

```diff
@@ -0,0 +1,4 @@
+---
+---
+
+chore: update issue templates
```

Comment:
Is there any way to see these templates in action? (rendered/visually)

Reply:
It isn't pretty/complete, but [yes](https://github.com/FuelLabs/fuels-ts/blob/db/chore/issue-templates/.github/ISSUE_TEMPLATE/1-bug.yaml).

Reply:
Thanks.